rm/rm-server/build.gradle

```diff
@@ -21,8 +21,8 @@ dependencies {
     testCompile "org.objectweb.proactive:programming-extension-pnpssl:${programmingVersion}"
     testCompile project(':common:common-api').sourceSets.test.output
 
-    testCompile 'org.jruby:jruby-complete:9.0.5.0'
-    testCompile 'org.python:jython-standalone:2.7.0'
+    testCompile 'org.jruby:jruby-complete:9.1.12.0'
+    testCompile 'org.python:jython-standalone:2.7.1'
```

Comment:
Apparently it has been released recently, so it should not be a beta any more:
http://forum.imagej.net/t/jython-2-7-1-released/5943